### ISSUE TITLE
Order History : ContextErrorException 

### ISSUE THREAD

**Issue by EGL59** [PrestaShop/PrestaShop#10040]
Hello,

Since the migration PrestaShop V1.6.1.20 to PrestaShop 1.7.4.2 no "old customer" can view their order history when connected course.

Here is the message when you click on the button "History and details of my orders" :

(1/1) ContextErrorException 
Notice: Undefined index: 
in Order.php line 925 
at OrderCore::getCustomerOrders(1)in HistoryController.php line 72 
at HistoryControllerCore->getTemplateVarOrders()in HistoryController.php line 55 
at HistoryControllerCore->initContent()in Controller.php line 255 
at ControllerCore->run()in Dispatcher.php line 428 
at DispatcherCore->dispatch()in index.php line 28 

New customers, ie those created after migration have no problem to view their order history.

Does anyone have an idea of the origin of this issue.
I have no module installed, I use the default theme.

After a long search, I find no solution.

**Additionnal information**
Version de PrestaShop : 1.7.4.2 
Version de PHP : 7.2.4
Version de MySQL : 5.7.21
Version du logiciel serveur : Apache/2.4.33 (Win64) PHP/7.2.4

Best regards,

**Comment by khouloudbelguith** [comment]
Hi @EGL59,

What is the exact version of the 1-click upgrade did you use?
PS: English is the norm on the Github 
Thank you!

Best regards, Khouloud

**Comment by EGL59** [comment]
Hi khouloudbelguith,

I don't use "the 1-click upgrade" because I had too much trouble and error message when I wanted to use this module. Still, it has worked well for all version V1.6.x.

But this time, for the migration to version 1.7.4.2, I do everything step by step…

Everything is good for me, after many corrections in the database, except this last point to consult the history of orders.

Can not fix this issue :(

**Comment by khouloudbelguith** [comment]
Hi @EGL59,

Thanks to follow this link: https://devdocs.prestashop.com/1.7/basics/keeping-up-to-date/upgrade/ for more details about the upgrade.
Thanks to check & feedback.

Best regards, Khouloud

**Comment by EGL59** [comment]
Hi khouloudbelguith,

Thank you for this link, but I have already all check

I tried more methods found on the internet. None are working properly.
I have never had a problem for all updates with V1.6.x.

But this time, it is not possible to do it automatically.

I only have one problem left! Just one !!!!
I have already solved a lot of problems for this migration.

But here I have no idea. What does this message mean?
This is a known error message, but nobody gives a solution.

Can you do something for this particular problem?
I am not the only one with this error message

Best regards,

**Comment by khouloudbelguith** [comment]
Hi @EGL59 ,

This issue occurs only with one old customer or with all oldest customers?

Best regards, Khouloud

**Comment by EGL59** [comment]
Hi @khouloudbelguith,

With all oldest customers.

I tested three, so I deduce that it is done with all oldest. (before migration).
New customers with 1.7 version have no problem.


Best regards,

**Comment by khouloudbelguith** [comment]
Hi @EGL59,

Despite our several trials, we could not reproduce your issue with the provided information.
It seems that your issue is not a PrestaShop's core bug but most likely a server configuration or customization problem.
I invite you to contact our support team: http://addons.prestashop.com/en/388-support to help you solve your issue.

Best regards, Khouloud

**Comment by EGL59** [comment]
Hi @khouloudbelguith,

I am surprised at your answer.
That does not really reassure me.
We are in the context of a Prestashop 1.6.1.20 migration to Prestashop 1.7.4.2.

None of the methods work properly, there is always a problem that no longer allows the use of our e-commerce site.

We had to do everything manually and here we have a last problem that no one knows how to solve.

Our server and our installation works perfectly well and we used the Classic theme without any modules.

This is very disturbing.
Is there anyone who can answer us here?

Otherwise, we will not do the migration to Prestashop 1.7.4.2

Best regards,
